### PR TITLE
Adding Sequelize support

### PR DESCRIPTION
**Pull request by stjohnjohnson**
Not much here, just exposing the right options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ This API comes preloaded with 4 (four) resources:
 Two (2) options for datastores:
```
Comment (d2lam) [inline README.md]: Three

File: config/custom-environment-variables.yaml [hunk 2]
```diff
@@ -38,6 +38,19 @@ httpd:
 
 datastore:
     plugin: DATASTORE_PLUGIN
+    sequelize:
+        # Type of server to talk to
+        dialect: DATASTORE_SEQUELIZE_DIALECT
+        # Database name
+        database: DATASTORE_SEQUELIZE_DATABASE
```
Comment (d2lam) [inline config/custom-environment-variables.yaml]: Do you want to add a default option for database?

Comment (d2lam) [inline config/custom-environment-variables.yaml]: Nvm it's already default here: https://github.com/screwdriver-cd/datastore-sequelize/blob/master/index.js#L126

File: README.md (hunk 1 above)
Comment (stjohnjohnson) [inline README.md]: Fixed

**Pull request merged**